src/components/switch/switch.tsx

```diff
@@ -12,6 +12,13 @@ import {
 import { createRandomString } from '../../util/random-string';
 
 /**
+ * The Switch component is a fundamental element in UI design that serves as a toggle switch
+ * to control the state of a specific setting or option in an application or website.
+ * The two distinct positions of the Switch are visually indicative of the two states:
+ * ON and OFF; making it easy for users to understand the current state of the controlled feature.
+ *
+ * The Switch component is widely used in user interfaces to enable users to
+ * quickly and intuitively change binary settings.
  * @exampleComponent limel-example-switch
```

Comment:
Maybe a bit more crm specific examples? 

Here are some situations where you might consider using a Switch component:
1. Settings and Preferences: Use a Switch component to allow users to enable or disable specific settings or 
preferences in your application. For example, you could use a Switch to let users toggle the "Dark Mode," 
"Notifications," or "Remember Me" options. 
2. Filters and Search Options: In search interfaces, you might use a Switch to enable or disable specific filters or 
search options. For example, you could use a Switch to let users toggle between "All" and "Mine" modes.

Reply:
But Lime Elements is not only for the CRM, so we cannot refer to things that require the reader to have a background knowledge of the CRM. We need to be more generic. Also "Dark Mode," "Notifications," or "Remember Me" options could be visualized either by using a switch or checkbox, depending on the implementation. There is nothing that says they must be _only_ done by using a switch.